docs/examples/custom/bottom-toolbar-editor.md

```diff
@@ -14,4 +15,422 @@ Additionally, thanks to the flexibility offered by the {@link framework/guides/a
 
 The presented combination of the UI and editor's features works best for integrations where text creation comes first and formatting is applied occasionally, for example in email applications, (forum) post editors, chats or instant messaging. You can probably recognize this UI setup from some popular applications such as Gmail, Slack or Zendesk.
 
+## Demo
+
 {@snippet examples/bottom-toolbar-editor}
+
+## Editor example configuration
+
+<details>
+<summary>View editor configuration script</summary>
+
+```js
+
+/* globals console, window, document */
+
```

Comment:
We can remove those lines in both snippets as well.

Reply:
Removed in both.